controllers/movie.js

```diff
@@ -48,25 +54,34 @@ const readByGenre = async (req, res) => {
 
 const readByRecommendation = async (req, res) => {
   console.log(req.body);
+  const { certifications, genres, release, ratings, providers, foreign } = req.body;
```

Comment:
Providers is defined but not used right now. You can prob just remove it, unless you are planning to use it soon.